modules/ROOT/pages/apikit-asyncapi-module-reference.adoc

```diff
@@ -40,8 +40,8 @@ Default configuration
 | Name | Type | Description | Default Value | Required
 | Name | String | Name for this configuration. Connectors reference the configuration with this name. | None | Yes
 | Api Definition | String |  API definition. | None | Yes
-| Kafka Configs | Array of <<KafkaConfig>>. | None | No |  
-| Anypoint MQ Configs | Array of <<AnypointMqConfig>>. | None | No |  
+| Kafka Configs | Array of <<KafkaConfig>>. | Configuration settings for Kafka integration. | None | No  
```

Comment:
FYI - this is a list of kafka server configurations. One for each kafka server listed in the specification that was scaffolded to create the app.

Reply:
OK! Thanks!